DigitalLearningSolutions.Web/Controllers/TrackingSystem/CentreConfiguration/RegistrationPromptsController.cs

```diff
@@ -142,5 +148,59 @@ private void IgnoreAddNewAnswerValidation()
                 ModelState[key].ValidationState = ModelValidationState.Valid;
             }
         }
+
+        [HttpGet]
+        [Route("Add/SelectPrompt")]
+        public IActionResult AddRegistrationPromptSelectPrompt()

```

Comment:
I feel like it would be useful to list all the public methods/endpoints before the private methods in the file for a quick overview of all the endpoints 🤔